test/functional/tool_wallet.py

```diff
@@ -136,15 +136,15 @@ def test_invalid_tool_commands_and_args(self):
         self.assert_raises_tool_error('Error parsing command line arguments: Invalid parameter -foo', '-foo')
         self.assert_raises_tool_error('No method provided. Run `bitcoin-wallet -help` for valid methods.')
         self.assert_raises_tool_error('Wallet name must be provided when creating a new wallet.', 'create')
-        
```

Comment:
In bf62c6f50230ed11b0335bc254258701765c7f00 "typo: fix lint issue"

This commit should be squashed with the previous.